packages/components/src/components/teaser-media/teaser-media.stories.ts

```diff
@@ -18,242 +29,173 @@ export default {
-    return html`
-      <style>
-        #teaserWithContentPlaceholder::part(media) {
-          flex-grow: 1;
-        }
-      </style>
-      <div class="flex flex-wrap gap-8">
-        <sd-teaser-media variant="gradient-dark" class="min-w-[435px] max-w-4xl">
-          <div slot="media" class="relative">
-            <img
-              class="aspect-video object-cover"
-              src="./placeholders/images/architecture.jpg"
-              alt="A skyline of a city"
-            />
-          </div>
-          <div slot="meta" class="meta-info">
-            <span class="meta-info-item">01.12.2013</span>
-            <span class="meta-info-item">| Author name</span>
-          </div>
-          <h3 slot="headline">Not expandable teaser-media</h3>
-          <div class="flex flex-col gap-4">
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-            <div class="flex-none">
-              <sd-button href="#" target="_blank" variant="primary" inverted>Link</sd-button>
-            </div>
-          </div>
-        </sd-teaser-media>
-        <sd-teaser-media variant="gradient-white" class="min-w-[435px] max-w-4xl">
-          <div slot="media" class="relative">
-            <img
-              class="aspect-video object-cover"
-              src="./placeholders/images/architecture.jpg"
-              alt="A skyline of a city"
-            />
-          </div>
-          <div slot="meta" class="meta-info">
-            <span class="meta-info-item">01.12.2013</span>
-            <span class="meta-info-item">| Author name</span>
-          </div>
-          <h3 slot="headline">Expandable teaser-media</h3>
-          <div slot="expandable">
-            <p>
-              Expandable: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
-              labore et dolore magna aliqua.
-            </p>
-          </div>
-          <div class="flex flex-col gap-4">
-            <p>
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
-              dolore magna aliqua.
-            </p>
-            <div class="flex-none">
-              <sd-button href="#" target="_blank" variant="primary">Link</sd-button>
-            </div>
-            <div class="flex-none text-sm">
-              <p>@Copyright Lorem ipsum</p>
-            </div>
-          </div>
-        </sd-teaser-media>
-      </div>
-    `;
-  }
+/**
+ * `Copyright` can be shown at the bottom of the default slot.
+ */
+
+export const Copyright = {
+  name: 'Copyright (Optional)',
+  render: () => html`
+    <div style="margin-bottom: 40px; width: 600px">
+      <sd-teaser-media>
+        <img slot="media" src="./placeholders/images/architecture.jpg" class="aspect-video object-cover" alt="Test" />
+        <h3 slot="headline">Headline Media Teaser</h3>
+        <div class="flex flex-col sd-copyright" style="--copyright: '© Union Investment 2024'">
```

Comment:
align left
fill: "text.neutral-700";
typography: "body.normal.text-sm";

![image](https://github.com/user-attachments/assets/9d4dd22a-d7a8-4986-977b-4981fec4e87f)

Reply:
Not possible to overwrite styles to copyright component. Different issues have been created to add variants to it:

- [Orientation variant](https://github.com/solid-design-system/solid/issues/1246)
- [Text color](https://github.com/solid-design-system/solid/issues/1309)

Variants for removing padding, text shadow or position may be consider necessary.

FYI: @yoezlem and @coraliefeil

Reply:
@coraliefeil the styles have been updated as requested. However, I still believe that including copyright variants would be an important feature.